### PR TITLE
[MRG +1] Fixing small typos on data_preparation.rst

### PR DESCRIPTION
**Pull request by Naereen**
Fixing small typos on [doc/manipulating_images/data_preparation.rst](https://nilearn.github.io/manipulating_images/data_preparation.html).

### CODE REVIEW

**Comment by aabadie** [comment]
LGTM +1

**Comment by Naereen** [comment]
I have no idea why the circleci build failed, sorry.

**Comment by lesteve** [comment]
Looks like a download that didn't succeed in one of the examples:

```
Downloading data from https://www.nitrc.org/frs/download.php/8486/miyawaki2008.tgz ...
________________________________________________________________________________
../examples/02_decoding/plot_miyawaki_encoding.py is not compiling:
Traceback (most recent call last):
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/urllib/request.py", line 1240, in do_open
    h.request(req.get_method(), req.selector, req.data, headers)
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/http/client.py", line 1083, in request
    self._send_request(method, url, body, headers)
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/http/client.py", line 1128, in _send_request
    self.endheaders(body)
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/http/client.py", line 1079, in endheaders
    self._send_output(message_body)
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/http/client.py", line 911, in _send_output
    self.send(msg)
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/http/client.py", line 854, in send
    self.connect()
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/http/client.py", line 1229, in connect
    super().connect()
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/http/client.py", line 826, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/socket.py", line 711, in create_connection
    raise err
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/socket.py", line 702, in create_connection
    sock.connect(sa)
TimeoutError: [Errno 110] Connection timed out
```

**Comment by lesteve** [comment]
I kicked off another CircleCI build.

File: doc/manipulating_images/data_preparation.rst [hunk 1]
```diff
@@ -70,36 +70,35 @@ relevant for the research questions at hand.
...
+`transform()` can be called, with the data as argument, to perform some
+computation on data themselves (e.g., extracting time series from images).
 
 Note that the masker objects may not cover all the image transformations
-for specific tasks. Users who want to make some specific processing may
-have to call low-level functions (see e.g. :mod:`nilearn.signal`,
-:mod:`nilearn.masking`).
+for specific tasks. Users who want to make some specific processing
+may have to call low-level functions
+(see e.g., :mod:`nilearn.signal`, :mod:`nilearn.masking`).
```
Comment (lesteve) [inline doc/manipulating_images/data_preparation.rst]: It doesn't look like there is a strong agreement on whether e.g. should be followed by a comma or not, see for example [this](http://www.dailywritingtips.com/comma-after-i-e-and-e-g/).

I would leave it like it was previously.

File: doc/manipulating_images/data_preparation.rst [hunk 2]
```diff
@@ -70,36 +70,35 @@ relevant for the research questions at hand.
...
+to `scikit-learn <http://scikit-learn.org>`_\ 's transformers.
+First, objects are initialized with some parameters guiding
+the transformation (unrelated to the data). Then the `fit()` method
 should be called, possibly specifying some data-related
 information (such as number of images to process), to perform some
 initial computation (e.g., fitting a mask based on the data). Finally,
-transform() can be called, with the data as argument, to perform some
-computation on data themselves (e.g. extracting time series from images).
+`transform()` can be called, with the data as argument, to perform some
+computation on data themselves (e.g., extracting time series from images).
```
Comment (lesteve) [inline doc/manipulating_images/data_preparation.rst]: same comment about e.g. and comma.

File: doc/manipulating_images/data_preparation.rst [hunk 3]
```diff
@@ -70,36 +70,35 @@ relevant for the research questions at hand.
...
+On an advanced note, the underlying philosophy of these classes is similar
+to `scikit-learn <http://scikit-learn.org>`_\ 's transformers.
+First, objects are initialized with some parameters guiding
+the transformation (unrelated to the data). Then the `fit()` method
 should be called, possibly specifying some data-related
 information (such as number of images to process), to perform some
 initial computation (e.g., fitting a mask based on the data). Finally,
-transform() can be called, with the data as argument, to perform some
-computation on data themselves (e.g. extracting time series from images).
+`transform()` can be called, with the data as argument, to perform some
```
Comment (lesteve) [inline doc/manipulating_images/data_preparation.rst]: Not exactly sure what single backquotes do in sphinx even after reading [this](http://www.sphinx-doc.org/en/stable/markup/inline.html). Did you mean to use double-backquotes instead?

File: doc/manipulating_images/data_preparation.rst (hunk 1 above)
Comment (Naereen) [inline doc/manipulating_images/data_preparation.rst]: There is no formal rule, I agree, except consistency, and all the others e.g. were followed by a comma (in this file, and the rest of nilearn's documentation).

File: doc/manipulating_images/data_preparation.rst (hunk 3 above)
Comment (Naereen) [inline doc/manipulating_images/data_preparation.rst]: Well, I was not sure... Part of the file was using `*`code`*`, part was using backquotes...
I can update my PR and add double-blackquotes if you prefer.

**Comment by Naereen** [comment]
For the circle-ci build, *« Looks like a download that didn't succeed in one of the examples »* : so it does not have anything to do with my PR, right?

**Comment by lesteve** [comment]
> it does not have anything to do with my PR, right?

Another failure now, not related to your PR again. It'd be nice to be have CircleCI succeed in order to quickly look at the generated doc.

**Comment by Naereen** [comment]
So it worked now. Do you want any other change on my PR before merging?

File: doc/manipulating_images/data_preparation.rst (hunk 1 above)
Comment (lesteve) [inline doc/manipulating_images/data_preparation.rst]: > There is no formal rule, I agree, except consistency

My 2 cents: consistency about this kind of things is very hard if not impossible to maintain in the long run and the benefit/cost ratio of maintaining it is very low.

**Comment by lesteve** [comment]
> So it worked now. Do you want any other change on my PR before merging?

I haven't checked that the CircleCI documentation looked similar to the one of master, have you? You can look at the Artifacts tab in CircleCI for your PR and a master build and quickly check.

File: doc/manipulating_images/data_preparation.rst (hunk 1 above)
Comment (GaelVaroquaux) [inline doc/manipulating_images/data_preparation.rst]: I agree with @lesteve : I am almost always in favor of consistency, but for things like this, the benefit is very small.

**Comment by GaelVaroquaux** [comment]
> I haven't checked that the CircleCI documentation looked similar to the one of
> master, have you? You can look at the Artifacts tab in CircleCI for your PR and
> a master build and quickly check.

Non admin users don't have an "Artifacts" tab. It sucks a bit;

**Comment by aabadie** [comment]
FYI, here is the [link](https://circle-artifacts.com/gh/nilearn/nilearn/1421/artifacts/0/home/ubuntu/nilearn/doc/_build/html/index.html).
You can compare with current [master](https://circle-artifacts.com/gh/nilearn/nilearn/1414/artifacts/0/home/ubuntu/nilearn/doc/_build/html/index.html).

File: doc/manipulating_images/data_preparation.rst (hunk 1 above)
Comment (Naereen) [inline doc/manipulating_images/data_preparation.rst]: OK, I won't do tiny modifications like this in the future, sorry :sweat_smile:.

**Comment by Naereen** [comment]
The new doc on circle-ci's artifact looks as expected, LGTM.

File: doc/manipulating_images/data_preparation.rst [hunk 4]
```diff
@@ -453,16 +445,16 @@ A single seed is a sphere defined by the radius (in millimeters) and the
...
+Seeds can overlap spatially and are represented in a binary present/nonpresent
 coding (no weighting).
 Below is an example of a coordinates list of four seeds from the default mode network::
 
   >>> dmn_coords = [(0, -52, 18), (-46, -68, 32), (46, -68, 32), (0, 50, -5)]
 
-"radius" is an optional argument that takes a real value in millimeters.
-If no value is given for the "radius" argument, the single voxel at the given
+`radius` is an optional argument that takes a real value in millimeters.
+If no value is given for the `radius` argument, the single voxel at the given
```
Comment (banilo) [inline doc/manipulating_images/data_preparation.rst]: perhaps less ambiguous: "the radius will only contain the single voxel at the radius center"

File: doc/manipulating_images/data_preparation.rst [hunk 5]
```diff
@@ -428,14 +420,14 @@ keyword.
...
-file is given in the "maps_img" argument.
+file is given in the `maps_img` argument.
 
 One important thing that happens transparently during the execution of
 :meth:`NiftiMasker.fit_transform` is resampling. Initially, the images
-and the atlas do typically not have the same shape nor the same affine. Casting
-them into the same format is required for successful signal extraction
-The keyword argument `resampling_target` specifies which format (i.e.,
-dimensions and affine) the data should be resampled to.
+and the atlas do typically not have the same shape nor the same affine.
```
Comment (banilo) [inline doc/manipulating_images/data_preparation.rst]: "affine" only by itself sounds colloquial

File: doc/manipulating_images/data_preparation.rst [hunk 6]
```diff
@@ -144,15 +143,14 @@ Computing the mask
...
-one from the provided neuroimaging data.
-It is *very important* to verify the quality of the generated mask by
-visualization. This allows to see whether it
-is suitable for your data and intended analyses.
-Alternatively, the mask computation parameters can still be modified. See the
-:class:`NiftiMasker` documentation for a complete list of mask computation
-parameters.
+
+If a mask is not specified as an argument, :class:`NiftiMasker` will try to
+compute one from the provided neuroimaging data.
```
Comment (banilo) [inline doc/manipulating_images/data_preparation.rst]: perhaps add to the end of the sentence: "...neuroimaging data in a heuristic fashion."

Comment (Naereen) [inline doc/manipulating_images/data_preparation.rst]: @banilo I think you should do your own PR instead of suggesting/askg changes on this one.

**Comment by Naereen** [comment]
The checks passed, and I added your requests. Can you merge it?

**Comment by bthirion** [comment]
@Naereen for this you need two positive reviews.
Here is one: LGTM.
Many thanks !

**Comment by aabadie** [comment]
>you need two positive reviews. 

Here is mine ! merging, thanks !

**Comment by Naereen** [comment]
Thanks!
